### PR TITLE
[Time Boxing] Create Home Page Placeholder Component

### PR DESCRIPTION
**Pull request by RenaufalAlrafid**
Silahkan di review

### CODE REVIEW

**Review by irfandigalih — DISMISSED** [review]

**Comment by irfandigalih** [comment]
pal typo conatainer

File: lib/Scenes/Page/HomePage/Component/invitation_card.dart [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            'Ini Section Invitation Card',
+            style: TimeBoxingTextStyle.paragraph2(
+              TimeBoxingFontWeight.regular,
+              TimeBoxingColors.neutralBlack(),
+            ),
+          ),
+          const Spacer(),
+          const Icon(Icons.calendar_today),
+        ],
+      ), //JANGAN LUPA DIHAPUS BUAT PENANDA DOANG
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/invitation_card.dart]: is this comment still being used?

File: lib/Scenes/Page/HomePage/Component/today_task.dart [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+  @override
+  State<TimeboxingTodayTask> createState() => _TimeboxingTodayTaskState();
+}
+
+class _TimeboxingTodayTaskState extends State<TimeboxingTodayTask> {
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      // color: TimeBoxingColors.primary70(TimeBoxingColorType.tint), // harus dikasik state ada invitation apa enggak
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/today_task.dart]: same

File: lib/Scenes/Page/HomePage/Component/today_task.dart [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      color: TimeBoxingColors.neutralLotion(),
+      // padding: const EdgeInsets.fromLTRB(24, 8, 24, 8),
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/today_task.dart]: same

**Review by gilangpersada — DISMISSED** [review]

File: lib/Scenes/Page/HomePage/home_page.dart [hunk 4]
```diff
@@ -9,16 +12,40 @@ class HomePage extends StatefulWidget {
...
 }
+
+class TimeboxingHomePlaceholder {
+  String id;
+  bool isShow;
+  Widget widget;
+
+  TimeboxingHomePlaceholder(
+      {required this.id, required this.isShow, required this.widget});
+}
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/home_page.dart]: dipisahin file sendiri aja

**Review by gilangpersada — APPROVED** [review]

**Pull request merged**